fleet_manager/TaskStatus[] tasks
string assign_to_drone_id
---
bool success
